src/stores/CatalogStore.ts

```diff
@@ -144,15 +144,18 @@ export class CatalogStore {
                 yFraction = Math.PI / 180.0;
```

Comment:
I use degree as default unit, if the unit of the column is null in catalog file. Should we keep this way?